components/Tabs/src/MDCTabBar.m

```diff
@@ -292,6 +299,18 @@ - (void)setInkColor:(UIColor *)inkColor {
   }
 }
 
+- (UIColor *)inkColor {
+  return _inkColor;
+}
+
+- (void)setRippleColor:(UIColor *)rippleColor {
+  if (_rippleColor != rippleColor && ![_rippleColor isEqual:rippleColor]) {
```

Comment:
Please update this to an early-exit clause, similar to the logic below in `setEnableRippleBehavior:`

```objc
if (_rippleColor == rippleColor || [_rippleColor isEqual:rippleColor]) {
  return;
}
```